src/auctions.lua

```diff
@@ -0,0 +1,91 @@
+local Auction = {}
+
+-- Default Auction Settings
+AuctionSettings = {
+	durationMs = 60 * 1000 * 60 * 24 * 14, -- 14 days in milliseconds
+	decayRate = 0.000000000016847809193121693, -- 0.02037911 / durationMs
```

Comment:
Note that this is 27 digits after the decimal so keep that in mind if passing across process boundaries.